src/external/visualization_integration.cuh

```diff
@@ -3,19 +3,20 @@
 
 
 #include "../../lib/HTTPRequest/include/HTTPRequest.hpp"
+#include <string>
+#include <vector>
 
 #include "../simulation_objects/MapNode.cuh"
-#include <string>
 
 
 /**
  * Reads the first line from the file containing the visualization endpoint url and returns it
  *
  * Path to the file (relative to the executable's location) is `config/visualization_endpoint.txt`
  *
- * @returns Url to send data to be visualized to
+ * @returns std::pair<std::string, std::string> with each string being a url to send data to be visualized to
```

Comment:
This is nice but still needs more details (probably in the description rather than in the `@returns` directive - it's up to you).